test/functional/feature_tapscript_opcodes.py

```diff
@@ -379,5 +380,126 @@ def run_test(self):
         # Script which don't make sense should also pass
         self.tapscript_satisfy_test(CScript([OP_5, OP_INSPECTTX, OP_IF, OP_IF]))
 
+        # short handle to convert int to 8 byte LE
+        def le8(x, signed=True):
+            return int(x).to_bytes(8, 'little', signed=signed)
+
+        def check_add(a, b, c, fail=False):
+            self.tapscript_satisfy_test(CScript([OP_ADD64, OP_VERIFY, le8(c), OP_EQUAL]), inputs = [le8(a), le8(b)], fail=fail)
+        def check_sub(a, b, c, fail=False):
+            self.tapscript_satisfy_test(CScript([OP_SUB64, OP_VERIFY, le8(c), OP_EQUAL]), inputs = [le8(a), le8(b)], fail=fail)
+        def check_mul(a, b, c, fail=False):
+            self.tapscript_satisfy_test(CScript([OP_MUL64, OP_VERIFY, le8(c), OP_EQUAL]), inputs = [le8(a), le8(b)], fail=fail)
+        def check_div(a, b, q, r, fail=False):
+            self.tapscript_satisfy_test(CScript([OP_DIV64, OP_VERIFY, le8(q), OP_EQUALVERIFY, le8(r), OP_EQUAL]), inputs = [le8(a), le8(b)], fail=fail)
+        def check_le(a, b, res, fail=False):
+            self.tapscript_satisfy_test(CScript([OP_LESSTHAN64, res, OP_EQUAL]), inputs = [le8(a), le8(b)], fail=fail)
+        def check_leq(a, b, res, fail=False):
+            self.tapscript_satisfy_test(CScript([OP_LESSTHANOREQUAL64, res, OP_EQUAL]), inputs = [le8(a), le8(b)], fail=fail)
+        def check_ge(a, b, res, fail=False):
+            self.tapscript_satisfy_test(CScript([OP_GREATERTHAN64, res, OP_EQUAL]), inputs = [le8(a), le8(b)], fail=fail)
+        def check_geq(a, b, res, fail=False):
+            self.tapscript_satisfy_test(CScript([OP_GREATERTHANOREQUAL64, res, OP_EQUAL]), inputs = [le8(a), le8(b)], fail=fail)
+        def check_eq(a, b, res, fail=False):
+            self.tapscript_satisfy_test(CScript([OP_EQUAL64, res, OP_EQUAL]), inputs = [le8(a), le8(b)], fail=fail)
+        def check_lshift(a, l, res, fail=False):
+            self.tapscript_satisfy_test(CScript([l, le8(a), OP_LSHIFT64 , le8(res, False), OP_EQUAL]), fail=fail)
+        def check_rshift(a, l, res, fail=False):
+            self.tapscript_satisfy_test(CScript([l, le8(a), OP_RSHIFT64 , le8(res, False), OP_EQUAL]), fail=fail)
+        # Arithematic opcodes
+        self.log.info("Check Arethmetic opcodes")
```

Comment:
spelling :) `arithmetic`

Reply:
Ah, I see this is fixed two commits later

Reply:
Oops, I fixed this in the wrong commit. Changing it